api/envoy/api/v2/route/route.proto

```diff
@@ -169,6 +169,10 @@ message Route {
   // Route matching parameters.
   RouteMatch match = 1 [(validate.rules).message = {required: true}];
 
+  // [#not-implemented-hide:]
+  // If true, a filter will overwrite some or all of the RouteAction proto.
+  bool filter_overwrites_route_action = 17;
```

Comment:
I thought we had discussed adding a `FilterAction` into the `action` oneof below with an `Any`?

Reply:
Ah, okay -- that makes sense.  I think I misunderstood your suggestion earlier.

Is this more like what you had in mind?